shared/chat/conversation/info-panel/index.js.flow

```diff
@@ -1,14 +1,13 @@
 // @flow
 import {Component} from 'react'
-import {List} from 'immutable'
 
 type infoPanelProps = {
   muted: boolean,
   onAddParticipant: () => void,
   onMuteConversation: (muted: boolean) => void,
   onShowProfile: (username: string) => void,
   onToggleInfoPanel: () => void,
-  participants: List<{
+  participants: Array<{
```

Comment:
Fixing type error here

Reply:
Did this not show up as a type error?

Reply:
Yeap, it didn't show up as a type error. I was really confused for a while!